apps/allocations/app/components/Panel/NewBudget.js

```diff
@@ -46,7 +46,7 @@ class NewBudget extends React.Component {
       this.state.amount = BigNumber(props.editingBudget.amount)
         .div(ETH_DECIMALS)
       this.state.amountError = false
-      this.state.selectedToken = props.editingBudget.token.symbol === 'ETH' ? 0 : 1
+      this.state.selectedToken = 0
```

Comment:
It looks like this is what it was initialize to on line 42 above. I think we can safely delete this line.